### ISSUE TITLE
Difference among various pretrained huggingface models

### ISSUE THREAD

**Issue by liyouli666** [dmis-lab/biobert#182]
Hello, I am wondering the difference among various pretrained biobert models.
I want to use one of your pretrained biobert models to fine-tune SQuAD with [run_squad.py](https://github.com/huggingface/transformers/blob/main/examples/pytorch/question-answering/run_qa.py ).
However, as you can see from the following picture, there are many biobert models, I don't know which one should I choose.
![image](https://user-images.githubusercontent.com/26185859/194994479-7cacc1bc-9ea4-498a-8fcc-1b3a2b106cee.png)

Besides, as for the 3 different types, feature extraction, fill mask, token classification, if I use them for fine tuning a question answering task, will they introduce different influence?
